data/data/google/master/main.tf

```diff
@@ -0,0 +1,75 @@
+resource "google_compute_instance" "master" {
```

Comment:
You might want to consider adding a scheduling block to the master GCE instance. Something like this ...

```
scheduling {
    automatic_restart = "true"
    on_host_maintenance = "MIGRATE"
    preemptible = "false"
  }
```

By default the Terraform Google Cloud Provider will not enable `automatic_restart` if you don't explicitly set it. Looks like this will be be fixed in version 2.0 of the provider. See https://github.com/terraform-providers/terraform-provider-google/pull/2638 for details.